crates/interpreter/src/side_table.rs

```diff
@@ -42,7 +43,7 @@ pub struct Metadata<'m>(&'m [u8]);
 
 impl<'m> Metadata<'m> {
     pub fn type_idx(&self) -> usize {
```

Comment:
`type_idx` and `branch_table` also need to be polymorphic, because they are called from `valid.rs` too. But this can be done in another PR.